### ISSUE TITLE
testing

### ISSUE THREAD

**Issue by adnannaim** [adnannaim/naim#1]
<p>Issue details: Open <a href="https://accessibility.browserstack.com/site-scanner/projects/v2.interactivecares.com_NaimurDamo_9734179_a11y/builds?thBuildId=y1b2zecd0uagfxx3getsg8xniz2nzdxytzvxxfkx&recurring=true&tunnel_vision=false&tab=ALLISSUES&activeViolationId=5788&activeComponentId=p%23text-neutral-600.text-sm.leading-relaxed&activeBehaviorValue=&activeIssueId=369352915">URL</a> on Browserstack</p><br />Issue details<table><tr><th>Item&nbsp;&nbsp;</th><th>Data&nbsp;&nbsp;</th></tr><tr><td>Issue name&nbsp;&nbsp;</td><td>Elements must meet minimum color contrast ratio thresholds&nbsp;&nbsp;</td></tr><tr><td>Issue description&nbsp;&nbsp;</td><td>Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds&nbsp;&nbsp;</td></tr><tr><td>More Info on issue&nbsp;&nbsp;</td><td><a href="https://accessibility.browserstack.com/api/more-info/a11y-engine-4.0/color-contrast">https://accessibility.browserstack.com/api/more-info/a11y-engine-4.0/color-contrast</a>&nbsp;&nbsp;</td></tr><tr><td>Affected component&nbsp;&nbsp;</td><td>p.text-neutral-600.text-sm.leading-relaxed&nbsp;&nbsp;</td></tr><tr><td>WCAG tags&nbsp;&nbsp;</td><td>WCAG 1.4.3&nbsp;&nbsp;</td></tr><tr><td>Best Practice issue&nbsp;&nbsp;</td><td>No&nbsp;&nbsp;</td></tr><tr><td>Experimental&nbsp;&nbsp;</td><td>No&nbsp;&nbsp;</td></tr><tr><td>Needs review&nbsp;&nbsp;</td><td>No&nbsp;&nbsp;</td></tr><tr><td>Impact&nbsp;&nbsp;</td><td>Serious&nbsp;&nbsp;</td></tr><tr><td>Affected page&nbsp;&nbsp;</td><td><a href="https://v2.interactivecares.com/">https://v2.interactivecares.com/</a>&nbsp;&nbsp;</td></tr><tr><td>How to fix&nbsp;&nbsp;</td><td>Get more details here<a href="https://accessibility.browserstack.com/site-scanner/projects/v2.interactivecares.com_NaimurDamo_9734179_a11y/builds?thBuildId=y1b2zecd0uagfxx3getsg8xniz2nzdxytzvxxfkx&recurring=true&tunnel_vision=false&tab=ALLISSUES&activeViolationId=5788&activeComponentId=p%23text-neutral-600.text-sm.leading-relaxed&activeBehaviorValue=&activeIssueId=369352915">(https://accessibility.browserstack.com/site-scanner/projects/v2.interactivecares.com_NaimurDamo_9734179_a11y/builds?thBuildId=y1b2zecd0uagfxx3getsg8xniz2nzdxytzvxxfkx&amp;recurring=true&amp;tunnel_vision=false&amp;tab=ALLISSUES&amp;activeViolationId=5788&amp;activeComponentId=p%23text-neutral-600.text-sm.leading-relaxed&amp;activeBehaviorValue=&amp;activeIssueId=369352915)</a>&nbsp;&nbsp;</td></tr></table>
